Packs/RedLock/Integrations/RedLock/RedLock.py

```diff
@@ -359,6 +362,44 @@ def translate_severity(alert):
     return 0
 
 
+def get_remediation_details():
+    """
+    Retrieve remediation details for a given alert
+    """
+    alert_ids = argToList(demisto.getArg('alert-id'))
+    payload = {'alerts': alert_ids, 'filter': {}}
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+
+    if not alert_ids:
+        return_error('You must specify the alert-id to retrieve with CLI remediation')
+
+    response = req('POST', 'alert/remediation', payload, None)
+
+    if response:
+        for alert_id in alert_ids:
+            details = {
+                'ID': alert_id,
+                'Remediation': {
+                    'CLI': response['alertIdVsCliScript'][alert_id],
+                    'Description': response['cliDescription']
+                }
+            }
+            context = {
+                'Redlock.Alert(val.ID == obj.ID)': details
+            }
+            MD = tableToMarkdown("remediationCLIoutput", details)
+            demisto.results({
```

Comment:
Don't call demisto.results for each alert. Instead call once for demisto.results for all the results of alerts

Reply:
Got it.  Moved this call outside the loop and adjusted accordingly.